src/main/java/filter/expression/Qualifier.java

```diff
@@ -243,6 +243,8 @@ public boolean isSatisfiedBy(IModel model, TurboIssue issue, MetaQualifierInfo i
             return satisfiesUpdatedHours(issue);
         case REPO:
             return satisfiesRepo(issue);
+        case COUNT:
+                return satisfiesCount();
```

Comment:
Sure, will move it to the meta qualifier list and add it to the `stripped` method!